### PR TITLE
fix(@desktop/wallet): Added squish test for change password and relogin

### LINKED ISSUE #6900: Can change password and relogin

### PR DESCRIPTION
**Pull request by Khushboo-dev-cpp**
fixes #6900

### What does the PR do

Squish test for user being able to change his password and logging in

### Affected areas

Settings

### Screenshot of functionality (including design for comparison)

- [x] I've checked the design and this PR matches it

### CODE REVIEW

File: test/ui-test/src/screens/SettingsScreen.py [hunk 1]
```diff
@@ -300,3 +308,20 @@ def check_backup_seed_phrase_workflow(self):
...
     def verify_seed_phrase_indicator_not_visible(self):
         verify_not_found(WalletSettingsScreen.BACKUP_SEED_PHRASE_BUTTON.value, "Check that backup seed phrase settings button is visible")
+        
+    def change_user_password(self, oldPassword: str, newPassword: str):
+        get_and_click_obj(ProfileSettingsScreen.CHANGE_PASSWORD_BUTTON.value)
+        
+        click_obj_by_name(ChangePasswordMenu.CHANGE_PASSWORD_CURRENT_PASSWORD_INPUT.value)
+        type(ChangePasswordMenu.CHANGE_PASSWORD_CURRENT_PASSWORD_INPUT.value, oldPassword)
+        
+        click_obj_by_name(ChangePasswordMenu.CHANGE_PASSWORD_NEW_PASSWORD_INPUT.value)
```
Comment (jrainville) [inline test/ui-test/src/screens/SettingsScreen.py:318]: Do we really need to click the input first? `type` should do the focus correctly.

**Review by jrainville — COMMENTED** [review]
Nice job. I just think you might not need the `click_obj_by_name` calls

**Review by stefandunca — APPROVED** [review]

**Review by noeliaSD — COMMENTED** [review]
Great job! Minor comments added.

I'm trying to run the test on my machine but having an issue bc the OS prompts me with a warning dialog saying that the desktop app is trying to authenticate me.. So then, the squish loses the focus and then it fails. Which permission desktop app is needing to do this action??

https://user-images.githubusercontent.com/97019400/186608058-a336ef35-c3c7-4882-a297-67410e18d469.mov

**Comment by Khushboo-dev-cpp** [comment]
> Great job! Minor comments added.
> 
> I'm trying to run the test on my machine but having an issue bc the OS prompts me with a warning dialog saying that the desktop app is trying to authenticate me.. So then, the squish loses the focus and then it fails. Which permission desktop app is needing to do this action??
> 
>  Screen.Recording.2022-08-25.at.09.52.55.mov

Hey just to note, this will only occur in case you have saved your password on macOs's keychain? so it shouldnt be something that should affect the normal tests on CI? or do we want to support this use case too?
@noeliaSD let me know what you think.

File: test/ui-test/src/screens/SettingsScreen.py (hunk 1 above)
Comment (Khushboo-dev-cpp) [reply test/ui-test/src/screens/SettingsScreen.py]: yep, not needed removed them!

**Review by jrainville — APPROVED** [review]

**Comment by noeliaSD** [comment]
> > Great job! Minor comments added.
> > I'm trying to run the test on my machine but having an issue bc the OS prompts me with a warning dialog saying that the desktop app is trying to authenticate me.. So then, the squish loses the focus and then it fails. Which permission desktop app is needing to do this action??
> > Screen.Recording.2022-08-25.at.09.52.55.mov
> 
> Hey just to note, this will only occur in case you have saved your password on macOs's keychain? so it shouldnt be something that should affect the normal tests on CI? or do we want to support this use case too? @noeliaSD let me know what you think.

I didn't save the password on macOs's keychain, just only ran the test and it was what happen, that's why I'm surprised with the result :thinking:

**Review by osmaczko — APPROVED** [review]

**Comment by Khushboo-dev-cpp** [comment]
> Great job! Minor comments added.
> 
> I'm trying to run the test on my machine but having an issue bc the OS prompts me with a warning dialog saying that the desktop app is trying to authenticate me.. So then, the squish loses the focus and then it fails. Which permission desktop app is needing to do this action??
> 
>  Screen.Recording.2022-08-25.at.09.52.55.mov

Iv removed this code in case the user has chosen to not save his password on keychain can someone please test it on macOS? @alaibe do you have mac os

**Comment by Khushboo-dev-cpp** [comment]
@noeliaSD once you are back can you please test this :)

**Comment by stefandunca** [comment]
> @noeliaSD once you are back can you please test this :)
I had the chance to test it and it fails for me



At first glance the real name lookup is too simple and might find a wrong listview  https://github.com/stefandunca/status-desktop/blob/38667e06a74691257516630da28370d5c544f0ad/test/ui-test/testSuites/global_shared/scripts/login_names.py#L9

**Comment by noeliaSD** [comment]
> @noeliaSD once you are back can you please test this :)

Tested!!! LGTM! :)

**Review by noeliaSD — APPROVED** [review]

**Pull request merged**